File: BE/models/LocationType.js
/**
 * Model đại diện cho loại địa điểm đi chơi
 */
export class LocationType {
  constructor(data = {}) {
    this.id = data.id || null;
    this.name = data.name || '';
    this.description = data.description || '';
    this.icon = data.icon || '';
  }
}

File: BE/src/main/java/com/food/app/model/LocationType.java
package com.food.app.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * Model đại diện cho loại địa điểm đi chơi
 * Ví dụ: Nhà hàng, Quán cà phê, Công viên, Bảo tàng, v.v.
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class LocationType {
    private Long id;
    private String name;
    private String description;
    private String icon; // Icon hoặc emoji đại diện cho loại địa điểm
}


